src/utils.ts

```diff
@@ -167,29 +167,32 @@ export function toHuman(
   x: BN,
   baseUnit: string,
   magnitude: number,
-  adjustMagnitude = 0,
+  adjustMagnitude = 0
 ): string {
-  const nomination = new BN(10).pow(new BN(magnitude));
+  let adjustment;
```

Comment:
One nit here.  I would use a ternary here `let adjustment = <bool> ? 0 : Math...`.  And make `x.toString().length` a local variable to make it more readable.

Reply:
sure, I'll edit it and push